src/services/post-install-utility.js

```diff
@@ -48,11 +52,13 @@ class PostInstallDisplayManager {
 
     if (this.hasProjects()) {
       console.warn(chalk.yellowBright(` » ${PORT_WARNING}`));
+      console.warn(chalk.yellowBright(` » ${AUTOCOMLETE_WARNING}`));
```

Comment:
Should we not display autocomplete warning for a fresh install with no profiles configured as well? I might be out of context here, but I think it shouldn't be check based as what I could see from the ask, and should display this autocomplete warning for every install.

Reply:
Nice spot! Made the change